app/service/case_api.py

```diff
@@ -24,3 +24,15 @@ def create_case(self, deed_id):
             return case_from_json(response.json())
         else:
             response.raise_for_status()
+
+    def update_case_with_deed(self, deed_id, case_id):
+        payload = {
+            "deed_id": deed_id
+        }
+        endpoint = self.case_endpoint + "/" + case_id + "/deed"
```

Comment:
Can you use string formatting over concatenation? It prevents some obscure type problems that can occur:

```
"{case}/{case_id}/deed".format(
    case=self.case_endpoint,
    case_id=case_id
)
```